packages/datatrak-web-server/src/routes/TasksRoute.ts

```diff
@@ -162,37 +162,14 @@ export class TasksRoute extends Route<TasksRequest> {
       page,
     });
 
-    const formattedTasks = tasks.map((task: SingleTask) => {
-      const {
-        entity_id: entityId,
-        'entity.name': entityName,
-        'entity.country_code': entityCountryCode,
-        'survey.code': surveyCode,
-        survey_id: surveyId,
-        'survey.name': surveyName,
-        ...rest
-      } = task;
-      return {
-        ...rest,
-        entity: {
-          id: entityId,
-          name: entityName,
-          countryCode: entityCountryCode,
-        },
-        survey: {
-          id: surveyId,
-          name: surveyName,
-          code: surveyCode,
-        },
-      };
-    });
+    const formattedTasks = tasks.map((task: TaskT) => formatTaskResponse(task));
```

Comment:
It's so nice that the single and plural endpoints are returning the same data type with this util